packages/scripts/src/create-webpack-configs.ts

```diff
@@ -43,27 +44,39 @@ export function createWebpackConfigs(options: ICreateWebpackConfigsOptions): web
         publicPath = '',
         createWebpackConfig,
         environments: { electronRendererEnvs, webEnvs, workerEnvs },
+        context,
     } = options;
 
     if (!baseConfig.output) {
         baseConfig.output = {};
     }
     baseConfig.output.publicPath = publicPath;
+
+    const packageJsonPath = fs.findClosestFileSync(__dirname, 'package.json'); // We want "our" package to extract the package name;
+    if (!packageJsonPath) {
+        // Should never happen
+        throw new Error('Cannot find package.json for determening cache dir');
+    }
+    const packageJson = fs.readJsonFileSync(packageJsonPath) as { name: string };
+
+    const projectCacheDir = findCacheDir({ name: packageJson.name, cwd: context, create: true });
```

Comment:
using the package name isn't ideal. it has / in it, as it's a scoped package.

Reply:
Besides, what is the problem within being scoped?
It just creates another nested directory, and package names guarantee uniqueness @AviVahl